frontend/src/scenes/session-recordings/player/PlayerSettings.tsx

```diff
@@ -0,0 +1,108 @@
+                                        { value: 'newer', label: 'Newer recordings' },
+                                        { value: 'older', label: 'Older recordings' },
+                                    ]}
+                                    size="small"
+                                />
+                            </div>
+                        ),
+                    },
+                    {
+                        custom: true,
+                        label: () => (
+                            <LemonSwitch
+                                className="px-2 py-1"
+                                checked={showMouseTail}
+                                onChange={setShowMouseTail}
+                                label="Show mouse tail"
+                                fullWidth
+                            />
+                        ),
+                    },
+                    {
+                        custom: true,
+                        label: () => (
+                            <LemonSwitch
+                                className="px-2 py-1"
+                                checked={showSeekbarPageviews}
+                                onChange={setShowSeekbarPageviews}
+                                label="Seekbar pageviews"
+                                fullWidth
+                            />
+                        ),
+                    },
+                    {
+                        custom: true,
+                        label: () => (
+                            <LemonSwitch
+                                handleContent={
+                                    <IconFastForward
+                                        className={clsx(
+                                            'p-0.5',
+                                            skipInactivitySetting ? 'text-primary-3000' : 'text-border-bold'
+                                        )}
+                                    />
+                                }
+                                fullWidth
+                                data-attr="skip-inactivity"
+                                className="px-2 py-1"
+                                checked={skipInactivitySetting}
+                                onChange={setSkipInactivitySetting}
+                                label="Skip inactivity"
+                            />
+                        ),
+                    },
+                    {
+                        label: `Playback speed (${speed}x)`,
+                        items: PLAYBACK_SPEEDS.map((speedToggle) => ({
+                            label: `${speedToggle}x`,
+                            onClick: () => setSpeed(speedToggle),
+                            active: speedToggle === speed,
+                        })),
+                        placement: 'right-end',
+                    },
+                ]}
+            >
+                <LemonButton size="small" icon={<IconEllipsis />} />
```

Comment:
Maybe this should be a gear icon but I'm undecided

Reply:
i think three dots is fine here..  leaves it open to add misc in one place